src/rollup/data/tracking/track-widget-action.js

```diff
@@ -52,13 +52,23 @@ export default function trackWidgetAction (action, widget, htmlElement) {
     pathforaDataObject.cancelledActions.push(params);
     break;
   case 'success.confirm':
-    params['pf-widget-action'] = !!widget.success && !!widget.success.confirmAction && widget.success.confirmAction.name || 'success confirm';
+    params['pf-widget-action'] = !!widget.formStates.success && !!widget.formStates.success.confirmAction && widget.formStates.success.confirmAction.name || 'success confirm';
```

Comment:
I think this will throw an error, but can `formStates` come through as `null` or `undefined`? We may want to do a check here `widget.formStates && ...` and elsewhere